packages/flutter_tools/test/commands.shard/hermetic/build_windows_test.dart

```diff
@@ -25,7 +25,8 @@ const String flutterRoot = r'C:\flutter';
 const String buildFilePath = r'C:\windows\CMakeLists.txt';
 const String buildUwpFilePath = r'C:\winuwp\CMakeLists.txt';
 const String visualStudioPath = r'C:\Program Files (x86)\Microsoft Visual Studio\2017\Community';
-const String cmakePath = visualStudioPath + r'\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe';
+const String _cmakePath = visualStudioPath + r'\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe';
+const String _defaultGenerator = 'Visual Studio 16 2019';
```

Comment:
Any reason for making these library-private, given it's a test?

Reply:
`cmakePath` as a default value for the named `cmakePath` parameter didn't work, and this seemed like a trivial way to disambiguate.